index.bs

```diff
@@ -4427,46 +4427,43 @@ user agent-defined value that provides human readable details of the error.
 The first is a <dfn id="dfn-simple-exception" export>simple exception</dfn>, which
-is identified by one of the following names:
-
-*   "<dfn exception><code>Error</code></dfn>"
-*   "<dfn exception><code>EvalError</code></dfn>"
-*   "<dfn exception><code>RangeError</code></dfn>"
-*   "<dfn exception><code>ReferenceError</code></dfn>"
-*   "<dfn exception><code>TypeError</code></dfn>"
-*   "<dfn exception><code>URIError</code></dfn>"
-
-These correspond to all of the [=ECMAScript error objects=] (apart from
-<emu-val>SyntaxError</emu-val>, which is deliberately omitted as
-it is for use only by the ECMAScript parser).
-The meaning of
-each [=simple exception=] matches
-its corresponding <emu-val>Error</emu-val> object in the
+is identified by one of the following type values:
+
+*   <emu-val>EvalError</emu-val>
+*   <emu-val>RangeError</emu-val>
+*   <emu-val>ReferenceError</emu-val>
+*   <emu-val>TypeError</emu-val>
+*   <emu-val>URIError</emu-val>
+
+These correspond to all of the [=ECMAScript error objects=]
+(apart from <emu-val>SyntaxError</emu-val> and <emu-val>Error</emu-val>,
+which are deliberately omitted as they are reserved for use
+by the ECMAScript parser and authors, respectively).
+The meaning of each [=simple exception=] matches
+its corresponding error object in the
 ECMAScript specification.
 
 The second kind of exception is a <dfn id="dfn-DOMException" interface export>DOMException</dfn>,
 which is an exception that encapsulates a name and an optional integer code,
 for compatibility with historically defined exceptions in the DOM.
 
-For [=simple exceptions=],
-the [=error name=] is the name
-of the exception.
-For a {{DOMException}},
-the [=error name=] must
-be one of the names listed in the [=error names table=]
-below.  The table also indicates the {{DOMException}}'s integer code
-for that error name, if it has one.
-
-There are two types that can be used to refer to
-exception objects: {{Error!!interface}}, which encompasses all exceptions,
+For [=simple exceptions=], the [=error name=] is the name of the exception.
+For a {{DOMException}}, the [=error name=] must be one of the names
+listed in the [=error names table=] below.
+The table also indicates the {{DOMException}}'s integer code for that error name,
+if it has one.
+
+There are two types that can be used to refer to exception objects:
+{{Error!!interface}}, which encompasses all exceptions,
 and {{DOMException}} which includes just DOMException objects.
 This allows for example an [=operation=]
 to be declared to have a {{DOMException}}
 [=return type=] or an [=attribute=]
 to be of type {{Error!!interface}}.
 
-Exceptions can be <dfn id="dfn-create-exception" for="exception" export>created</dfn> by providing its
-[=error name=].
+[=Simple exceptions=] can be <dfn id="dfn-create-exception" for="exception" export>created</dfn>
+by providing their [=error name=].
+{{DOMException|DOMExceptions}}, by providing their [=error name=] followed by {{DOMException}}.
```

Comment:
I remembered having a twitter thread about this grammar construction. Wasn't expecting @domenic to be the first reply, though. :D

I'll fix that.